package.json

```diff
@@ -65,7 +65,7 @@
     "@octokit/webhooks": "^10.6.2",
     "@octokit/webhooks-types": "^6.9.0",
     "@primer/react": "^35.14.1",
-    "@prisma/client": "^4.6.1",
+    "@prisma/client": "^4.16.1",
```

Comment:
revent the changes made in this file as they are not in the scope of this PR

Reply:
Thanks for the suggestion @yuval-hazaz. I have update the code as per your suggestion, Submitted PR